analysis_options.yaml

```diff
@@ -16,5 +17,4 @@ linter:
   rules:
     - await_only_futures
     - implementation_imports
-    - prefer_equal_for_default_values
```

Comment:
Is there a reason this was removed?

Reply:
It is in the imported file https://github.com/dart-lang/pedantic/blob/v1.1.0/lib/analysis_options.yaml#L17